base/init.sh

```diff
@@ -1,3 +1,16 @@
 #!/bin/bash
 
-pulpcore-manager createsuperuser --no-input --email admin@example.com || true
\ No newline at end of file
+pulpcore-manager createsuperuser --no-input --email admin@example.com || true
+
+if pulp >/dev/null
+then
+  if ! pulp --refresh-api status
+  then
+    pulp config create --overwrite --base-url "http://localhost:5001" --username "admin" --password "password"
```

Comment:
the nginx port, username and password are configurable. You can read these values from these environment variables which get passed into the container: https://github.com/pulp/oci_env/blob/main/client/oci_env/utils.py#L46

Reply:
Like this?